api/app/src/main/kotlin/packit/model/RunInfo.kt

```diff
@@ -13,7 +13,7 @@ class RunInfo(
     var taskId: String,
 
     var packetGroupName: String,
-    var status: String,
+    var status: String? = null,
```

Comment:
when is this null? im sure when we can give it the initial status of "PENDING" when we submit... and then when we fetch status this can/may be updated